### PR TITLE
Update Macro Definitions in WikiText.tid

### PR DESCRIPTION
**Pull request by twMat**
*The table is intended as an overview.
*The "Parameters as Variables" example is more clean cut than previously.

Please note that the text lacks an example of  direct `<<variable>>` invocation (because I'm not even sure I categorize or name this properly). Also my changes may contain factual errors or faulty lingo - please check!

### CODE REVIEW

File: editions/tw5.com/tiddlers/wikitext/Macro Definitions in WikiText.tid [hunk 1]
```diff
@@ -19,7 +19,17 @@ Alternatively, the entire definition can be presented on a single line, without
 
 A more formal [[presentation|Macro Definition Syntax]] of this syntax is also available.
 
-!! Placeholders
+!!Accessing variables and parameters
```
Comment (pmario) [inline editions/tw5.com/tiddlers/wikitext/Macro Definitions in WikiText.tid]: `!!<space>Accessing ...

Comment (twMat) [reply editions/tw5.com/tiddlers/wikitext/Macro Definitions in WikiText.tid]: Thanks. Forgive my ignorance of github but does this mean you've already made that change or is it only being pointed out and I'm supposed to make the modification somehow?

Comment (pmario) [reply editions/tw5.com/tiddlers/wikitext/Macro Definitions in WikiText.tid]: I just wanted to point it out. You can change it

Comment (pmario) [reply editions/tw5.com/tiddlers/wikitext/Macro Definitions in WikiText.tid]: just edit the file and commit it. It will be automatically added to this PR.

**Comment by Jermolene** [comment]
Thanks @twMat

**Pull request merged**